Events.php

```diff
@@ -57,13 +58,15 @@ public static function onTopMenuInit($event)
             return;
         }
 
-        $event->sender->addItem([
-            'label' => Yii::t('MailModule.base', 'Messages'),
-            'url' => Url::to(['/mail/mail/index']),
-            'icon' => '<i class="fa fa-envelope"></i>',
-            'isActive' => (Yii::$app->controller->module && Yii::$app->controller->module->id == 'mail'),
-            'sortOrder' => 300,
-        ]);
+        if(!ConfigureForm::getModule()->showInTopNav()){
```

Comment:
Would be easier when showInTopNav would return true instead of false (see other comment)